src/families/stellar/bridge/libcore.js

```diff
@@ -0,0 +1,296 @@
+  let totalSpent = !useAllAmount
+    ? amount.plus(estimatedFees)
+    : a.balance.minus(baseReserve);
+
+  if (useAllAmount) {
+    warnings.amount = new NotEnoughSpendableBalance("", {
+      minimumAmount: `${baseReserve.toString()} XLM`
+    });
+  }
+
+  if (
+    !errors.amount &&
+    amount
+      .plus(estimatedFees)
+      .plus(baseReserve)
+      .gt(a.balance)
+  ) {
+    errors.amount = new NotEnoughBalance();
+  }
+
+  if (
+    !errors.recipient &&
+    !errors.amount &&
+    (amount.lt(0) || totalSpent.gt(a.balance))
+  ) {
+    errors.amount = new NotEnoughBalance();
+    totalSpent = BigNumber(0);
+    amount = BigNumber(0);
+  }
+
+  if (!errors.amount && amount.eq(0)) {
+    errors.amount = new AmountRequired();
+  }
+
+  // if amount < 1.0 you can't
+  if (
+    !errors.amount &&
+    !(await checkRecipientExist({ account: a, recipient: t.recipient })) &&
+    amount.lt(10000000)
+  ) {
+    errors.amount = new NotEnoughBalanceBecauseDestinationNotCreated("", {
+      minimalAmount: "1 XLM"
+    });
+  }
+
+  if (t.memoType && t.memoValue && !isMemoValid(t.memoType, t.memoValue)) {
+    errors.transaction = new StellarWrongMemoFormat();
+  }
+
+  return Promise.resolve({
+    errors,
+    warnings,
+    estimatedFees,
+    amount,
+    totalSpent
+  });
+};
+
+const prepareTransaction = async (a, t) => {
+  const networkInfo = t.networkInfo || (await getAccountNetworkInfo(a));
+  invariant(networkInfo.family === "stellar", "stellar networkInfo expected");
+
+  const fees = t.fees || networkInfo.fees;
+  const baseReserve = t.baseReserve || networkInfo.baseReserve;
+
+  const getMemoData = async () => {
+    if (t.memoType) {
+      return {
+        memoType: t.memoType,
+        memoTypeRecommended: t.memoTypeRecommended
+      };
+    } else {
+      const { recipientError } = await validateRecipient(
+        a.currency,
+        t.recipient
+      );
+      if (!recipientError) {
+        const memoType = await memoTypeCheck(t.recipient);
+        const memoTypeRecommended = memoType !== null ? true : false;
+        return { memoType, memoTypeRecommended };
+      }
+      return {
+        memoType: undefined,
+        memoTypeRecommended: false
+      };
+    }
+  };
+
+  const { memoType, memoTypeRecommended } = await getMemoData();
+
+  if (
+    t.networkInfo !== networkInfo ||
+    t.fees !== fees ||
+    t.baseReserve !== baseReserve ||
+    t.memoType !== memoType
+  ) {
+    return {
+      ...t,
+      networkInfo,
+      fees,
+      baseReserve,
+      memoType,
+      memoTypeRecommended
+    };
+  }
+
+  return t;
+};
+
+const estimateMaxSpendable = async ({
+  account,
+  parentAccount,
+  transaction
+}) => {
+  const mainAccount = getMainAccount(account, parentAccount);
+  const t = await prepareTransaction(mainAccount, {
+    ...createTransaction(),
+    ...transaction,
+    useAllAmount: true,
+    recipient: notCreatedStellarMockAddress // not used address
```

Comment:
can you move this **before** the `...transaction` ? it's a mistake i did on all other coins that i'm about to send a fix for. :sorry: